SUPPORTED_SCENARIOS.md

```diff
@@ -77,5 +79,9 @@ This option requires packages based on the architecture of the system:
 * arm architecture requires: <b>grub2-arm-efi</b>.
 * aarch64 architecture requires: <b>grub2-arm64-efi</b>.
 
+## systemd-boot
+If you're running a multiboot EFI system, systemd-boot can provide easier boot management and may even reduce your boot times.
+Systemd-boot will be supported on x86_64 EFI architecture only.
```

Comment:
Just to note that should work on aarch64 too if EFI is used via uboot (like grub2 today)

Reply:
Currently we are supporting x86_64 only. But good point....